### PR TITLE
feat: add v0.35.3 testnet upgrade to nav

### PR DESCRIPTION
**Pull request by ffe9f8**

### CODE REVIEW

File: src/pages/resources/testnet/upgrades/v35-3.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Testnet upgrade: v0.35
+# Testnet upgrade: v0.35.3
```
Comment (milapsheth) [inline src/pages/resources/testnet/upgrades/v35-3.mdx:1]:
```suggestion
# Testnet upgrade: v0.35
```
The patch version doesn't need to be mentioned in the title, similar to past upgrades. Makes it seem like all patch releases are consensus breaking

File: src/layouts/navigation.ts [hunk 2]
```diff
@@ -533,6 +533,10 @@ export const getNavigation = (section) => {
                     title: "v0.34",
                     href: "/resources/testnet/upgrades/v34",
                   },
+                  {
+                    title: "v0.35.3",
+                    href: "/resources/testnet/upgrades/v35-3",
```
Comment (milapsheth) [inline src/layouts/navigation.ts:538]:
```suggestion
                    href: "/resources/testnet/upgrades/v35",
```

File: src/layouts/navigation.ts [hunk 3]
```diff
@@ -533,6 +533,10 @@ export const getNavigation = (section) => {
                     title: "v0.34",
                     href: "/resources/testnet/upgrades/v34",
                   },
+                  {
```
Comment (milapsheth) [inline src/layouts/navigation.ts:536]: can you change this to descending order? latest upgrades should show up first

**Comment by ffe9f8** [comment]
Moved changes to [PR 808](https://github.com/axelarnetwork/axelar-docs/pull/808).

**Pull request closed without merging**